mmpose/codecs/simcc_label.py

```diff
@@ -105,15 +105,16 @@ def encode(
                 f'{self.smoothing_type}. Should be one of '
                 '{"gaussian", "standard"}')
 
-    def decode(self,
-               encoded: Tuple[np.ndarray,
-                              np.ndarray]) -> Tuple[np.ndarray, np.ndarray]:
+    def decode(self, simcc_x: np.ndarray,
```

Comment:
@Tau-J In this PR I slightly changed the decoding process in the base head. Consequently, `codec.decode()` takes packed arguments instead of a single tuple. Is that ok with you?

Reply:
No problem